code/dist/style.css

```diff
@@ -0,0 +1,272 @@
+    font-weight: bold;
+    float: right;
+    margin-left: 5px;
+  }
+  
+  .active:after {
+    content: "\2212";
+  }
+  
+  .panel {
+    padding: 0 20px;
+    background-color: white;
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.2s ease-out;
+  }
+
+    /* The container */
+
+    .form {
+      width: 450px;
+      padding: 20px;
+      background-color: #eee;
+    }
+
+    .container {
+      display: block;
+      position: relative;
+      padding-left: 35px;
+      margin-bottom: 12px;
+      cursor: pointer;
+      font-size: 18px;
+      -webkit-user-select: none;
+      -moz-user-select: none;
+      -ms-user-select: none;
+      user-select: none;
+    }
+    
+    /* Hide the browser's default radio button */
+    .container input {
+      position: absolute;
+      opacity: 0;
+      cursor: pointer;
+    }
+    
+    .checkmark {
+      position: absolute;
+      top: 0;
+      left: 0;
+      height: 25px;
+      width: 25px;
+      background-color: #5B5B5B;
+      border-radius: 50%;
+    }
+    
+    .container:hover input ~ .checkmark {
+      background-color: #ccc;
+    }
+    
+    .container input:checked ~ .checkmark {
+      background-color: #FC9D9D;
+    }
+    
+    .checkmark:after {
+      content: "";
+      position: absolute;
+      display: none;
+    }
+    
+    .container input:checked ~ .checkmark:after {
+      display: block;
+    }
+    
+    .container .checkmark:after {
+      top: 9px;
+      left: 9px;
+      width: 8px;
+      height: 8px;
+      border-radius: 50%;
+      background: white;
+    }
+    
+    input[type=text] {
+      width: 100%;
+      padding: 12px;
+      margin: 10px 0;
+      border: 1px solid #ccc;
+      box-sizing: border-box;
+      cursor: pointer;
+    }
+    
+    input[type=email] {
+      width: 100%;
+      padding: 12px;
+      margin: 10px 0;
+      border: 1px solid #ccc;
+      box-sizing: border-box;
+      cursor: pointer;
+    }
+    
+    input[type=submit] {
+      width: 100%;
+      background-color: #FC9D9D;
+      color: white;
+      padding: 14px 20px;
+      margin: 0 0 10px 0;
+      border: 1px solid #ccc;
+      cursor: pointer;
+    }
+    
+    textarea {
+      width: 100%;
+      height: 100px;
+      padding: 12px 20px;
+      margin: 10px 0;
+      box-sizing: border-box;
+      border: 1px solid #ccc;
+      background-color: #FFFFFF;
+      cursor: pointer;
+      resize: none;
+      color: #5B5B5B ;
+      font-family: Helvetica, Arial, sans-serif;
+      font-size: small;
+    }
+    
+
+  .button {
+    appearance: none;
+    background-color: #FC9D9D;
+    border: none;
+    font-family: Helvetica, Arial, sans-serif;
+    font-size: small;
+    margin-top: 20px;
+    padding: 10px;
+    justify-self: start;
+  }
+
+  .square_btn{
+    display: inline-block;
+    padding: 0.5em 1em;
+    text-decoration: none;
+    background: #FC9D9D;
+    border-left: solid 6px #5B5B5B;
+    color: #FFFFFF;
```

Comment:
Btw, small side note - if we define color by hexadecimal approach and all characters are the same (as in `#FFFFFF`), we can make it shorter :

```
color: #FFF;
```

Reply:
Noted!